### ISSUE TITLE
It is not working

### ISSUE THREAD

**Issue by HenriqueKibe** [DarkLord76865/Autofish-Fishing-Planet#7]
I am trying to make it work but it does not. I do not know if I need to do something else. I just downloaded the last version of it and opened it. It doesnt work.
Should I do something different than just opening the application? Do I need Pythong to make it work?

OBS: I am using more than 1 monitor on my setup with different resolution (1920x1080 and 3440x1440)

If you want, add me on Discord to help me HenriqueKibe#5181

**Comment by DarkLord76865** [comment]
Hi,
can you give me more information about what exactly doesn't work? Does the app not work/crash, or is the bot not working? App should work regardless of your monitors, but I don't know about bot. It might or might not work. What platform are you on, is it Windows 10? Maybe your antivirus is blocking it.

**Comment by HenriqueKibe** [comment]
The app opens but when I click Alt + X nothing happens. That little circle on the left side turns green and nothing happens. I am using Windows 11 and tried with antivirus on and off.

**Comment by DarkLord76865** [comment]
Do you have Fishing Planet open? You should open the app, then open Fishing Planet, go to the lake you want, setup rod and lure you want, and then, with Fishing Planet full screen (without seeing the app) press alt+x to start the bot. The little circle that turns green is more for debugging purposes, you don't usually look at it as the app window is in the background with Fishing Planet in the foreground. If you are already doing all of that, then it might be that your dual monitor setup is the problem. If that is the case, I can't unfortunately help you. I only have one monitor so there is no way for me to test, and fix the app for multi monitor setup. Sorry. :sad:

**Comment by DarkLord76865** [comment]
Do you have Fishing Planet open? You should open the app, then open Fishing Planet, go to the lake you want, setup rod and lure you want, and then, with Fishing Planet full screen (without seeing the app) press alt+x to start the bot. The little circle that turns green is more for debugging purposes, you don't usually look at it as the app window is in the background with Fishing Planet in the foreground. If you are already doing all of that, then it might be that your dual monitor setup is the problem. If that is the case, I can't unfortunately help you. I only have one monitor so there is no way for me to test, and fix the app for multi monitor setup. Sorry.

**Comment by HenriqueKibe** [comment]
Hey!!!
I just made it work here! It is all working fine now. However I just realized something is happening when I try to turn it on. Sometimes it works, sometimes it doesn’t. Did not figured it out whats heppening.

BTW, is it possible in a future update, make something to release a fish that is havier than the net? I got a fish that does not fit in my net and I got stuck on that screen with the fish and I had to return the fish manually!
